src/nativeapm/config/theme.ts

```diff
@@ -0,0 +1,269 @@
+/// <reference path="../references.ts" />
+
+type NativeApmThemeConfigType = {
+  wrapper: Record<string, any>;
+  spinner: Record<string, any>;
+  logo: Record<string, any>;
+  buttons: Record<string, any>;
+  form: Record<string, any>;
+  message: Record<string, any>;
+  actionImage: Record<string, any>;
+};
+
+const defaultTheme: NativeApmThemeConfigType = {
```

Comment:
you should use always rem and not px. Rems are relative units that are based on the document's font-size so, if the user zoom in or zoom out, the size of your elements will follow

Reply:
Fixed :)